seos_profile.c

```diff
@@ -132,5 +133,29 @@ bool ble_profile_seos_tx(FuriHalBleProfileBase* profile, uint8_t* data, uint16_t
         return false;
     }
 
-    return ble_svc_seos_update_tx(seos_profile->seos_svc, data, size);
+
+    uint16_t num_chunks = size / 19;
+    if (size % 19) num_chunks++;
+
+    uint8_t chunk[20];
```

Comment:
So I wasn't limiting the size before and it seemed to work, what was the situation where it wasn't?  I am familiar with BLE having the 20 byte limit in one of the directions (i've written code just like this before to chunk), I just can't recall which direction.  

Can you make a #define for 20?